src/server/app.py

```diff
@@ -1,11 +1,48 @@
-from flask import Flask
+import json
+
+from flask import Flask, request, Response
 from flask_cors import CORS
 
+import src.server.cea_608_encoder.byte_pair_generator as encoder
+
+
 app = Flask(__name__)
 CORS(app)
 
 
 @app.route("/submit", methods=['POST'])
 def submit():
-    pass
+    if request.get_json() is None:
+        app.logger.error('Received request with a mimetype other than JSON')
+        return Response(json.dumps({'Error': 'The endpoint requires a JSON mimetype: \'application/json\''}),
+                        status=415,
```

Comment:
Your ticket says that doing
`curl -d '{"key1":"value1", "key2":"value2"}' -H "Content-Type: application/xml" -X POST http://localhost:{port_number}/submit`
Should result in a 400, it results in this error instead. Here is the feedback from the server
`[2019-10-21 18:15:18,883] ERROR in app: Received request with a mimetype other than JSON
127.0.0.1 - - [21/Oct/2019 18:15:18] "POST /submit HTTP/1.1" 415 -
{"Error": "The endpoint requires a JSON mimetype: 'application/json'"}`